ihmc-avatar-interfaces/src/test/java/us/ihmc/avatar/DRCBumpyAndShallowRampsWalkingTest.java

```diff
@@ -41,6 +35,11 @@
 import us.ihmc.yoVariables.variable.YoBoolean;
 import us.ihmc.yoVariables.variable.YoDouble;
 
+import java.util.ArrayList;
+import java.util.Random;
+
+import static us.ihmc.robotics.Assert.*;
```

Comment:
Please don't use this, it's a deprecated set of tools.

Reply:
This was an accidental organize imports. I didn't add any calls to this.